server/src/Controller/Channel/ChannelController.js

```diff
@@ -0,0 +1,99 @@
+import {
+  create,
+  deleteDoc,
+  findOne,
+  findMany,
+  update,
+} from '../../Repositories/genericRepository';
+import successHandler from '../../libs/routes/successHandler';
+import { channelModel } from '../../Model';
+
+
+const getChannel = async (req, res, next) => {
+  try {
```

Comment:
Can we abstract this try catch? as this is used everywhere. 
Following is one of the ways, we can look for other ways. This can be added in common utils which can be used in both client and server
https://nunoalexandre.com/2016/12/20/encapsulating-try-catch-statements
@nimish-gupta @rakeshtembhurne @radhey1851 - Are you aware of some standard way of doing this?

Reply:
You can wrap a function around try catch, which will return an array like [success, error] similar to callback style.